huaweicloud/utils/tags.go

```diff
@@ -68,6 +68,21 @@ func TagsToMap(tags []tags.ResourceTag) map[string]string {
 	return result
 }
 
+// FlattenTagsToMap returns the list of tags into a map.
+func FlattenTagsToMap(tags interface{}) map[string]string {
```

Comment:
can the type be **[]interface{}**, then line 76 can removed

Reply:
type interface is more convenient.